src/core/component.js

```diff
@@ -251,15 +255,28 @@ Component.prototype = {
  */
 module.exports.registerComponent = function (name, definition) {
   var NewComponent;
+  var prevSibling;  // Previous sibling of current script.
```

Comment:
`previousSibling`? Let's avoid acronyms and contractions unless they're conventional wisdom